tests/checkpoint/test_checkpoint.py

```diff
@@ -127,7 +127,7 @@ def test_basic_checkpoint_config_validation(
     caplog,
     capsys,
 ):
-    context: FileDataContext = empty_data_context_stats_enabled
+    data_context: FileDataContext = empty_data_context_stats_enabled
```

Comment:
These changes are only beautification focused and correcting typographical errors.  No logic has been changed.